src/helm/impress/values.yaml

```diff
@@ -85,6 +85,16 @@ ingressCollaborationWS:
     nginx.ingress.kubernetes.io/proxy-send-timeout: "86400"
     nginx.ingress.kubernetes.io/upstream-hash-by: $arg_room
 
+ingressRedirects:
```

Comment:
You have to add the documentation before thisblock (take a look at the other) and then run 
```
$ cd src/helm/impress
$ ./generate-readme.sh
```